website/src/layouts/OrganismPage/OrganismPageLayout.astro

```diff
@@ -23,6 +27,7 @@ const lastUpdatedDate = await getLastUpdatedDate(getLapisUrl(view.organismConsta
     </div>
     <slot />
     <div class='flex justify-center' slot='secondary-footer'>
+        {downloadLinks.length > 0 && <AccessionsDownloadButton view={view} downloadLinks={downloadLinks} />}
         <span class='mr-4 p-2 text-xs'>Data last updated: {lastUpdatedDate ?? 'unknown'}</span>
```

Comment:
Not really this PR, but it would be nice to put this line into its own component, to have the same complexity as the other lines.